src/Monolog/Utils.php

```diff
@@ -31,7 +31,7 @@ public static function substr(string $string, int $start, ?int $length = null)
             return mb_strcut($string, $start, $length);
         }
 
-        return substr($string, $start, $length);
+        return substr($string, $start, $length ?: strlen($string));
```

Comment:
Probably should be a strict check for null, because otherwise passing a length of `0` (edge case for sure but eh..) would make it return the full string remnants instead of nothing.

Reply:
I changed the comparison.